### ISSUE TITLE
A bug with treetop in the quest

### ISSUE THREAD

**Issue by AlexMarsev666** [LunaPixelStudios/Prominence#130]
### Describe the Bug

A quest with treetap crafting is not being performed. I threw it under myself, it's still not being executed. Version 1.20.1

### Reproduction Steps

1. Crafted
2. I threw it under me
3. Nothing was fulfilled

### Screenshots and Videos

![image](https://github.com/LunaPixelStudios/Prominence/assets/69976658/0aaae949-c328-4e0f-aa06-61f61afe5001)
![image](https://github.com/LunaPixelStudios/Prominence/assets/69976658/ecc067ef-62c8-4e73-b34b-9fd44c4596b5)
![image](https://github.com/LunaPixelStudios/Prominence/assets/69976658/ab45678f-a091-467a-9ccc-8d817b0c929a)

### Operating System

Win 10

### Minecraft Version

1.19

### Modloader

Fabric

### Modpack Version

0.7.0

### Optifine

I used Optifine

### Other Mods

No

**Comment by Saiisako** [comment]
i got the same bug help me

**Comment by AlexMarsev666** [comment]
The essence of the bug is that the "nbt" argument does not work correctly. I'm not sure that if you delete it everywhere, everything will work correctly

**Comment by AlexMarsev666** [comment]
This problem is not only with this quest

**Comment by Angelthree95** [comment]
I found a workaround for this type of questing bug:
1. Enter the edit mode inside the quest which is not working
![image](https://github.com/LunaPixelStudios/Prominence/assets/48697334/1bb7bf5b-a219-4c9f-818a-527a67a2b97c)
2. Copy task ID
![image](https://github.com/LunaPixelStudios/Prominence/assets/48697334/5bef19d9-1db0-46c8-8114-1a162f4c8fe3)
3. Go to Tasks section and delete the bugged task
![image](https://github.com/LunaPixelStudios/Prominence/assets/48697334/1daed8a8-a862-4854-9e1c-56a57394a13d)
4. Add a new task: "gather item" type and paste in the ID you copied from overview tab.
![image](https://github.com/LunaPixelStudios/Prominence/assets/48697334/ac1e9a46-5405-43c0-b5cb-8c98528d9d85)
5. Submit
6. Change the item type to the item you originally had to collect
![image](https://github.com/LunaPixelStudios/Prominence/assets/48697334/1c7ca211-5f94-4366-b1b0-65935afd6d02)
7. Save
8. Collect the item and profit
Sometimes the workaround works only after restarting the game (had that happen to me once so far for 3 quests I edited)

**Comment by nvb-uy** [comment]
Fixed in v0.8.0
